docker/nanotube/nanotube.conf

```diff
@@ -0,0 +1,9 @@
+cluster a34000
```

Comment:
I am not sure I understand the purpose of this file. It's a crelay config but looks like nanotube one. Could you please explain?

Reply:
It was a left over from coping crelay config. Removed.